osu.Game/Rulesets/Judgements/DrawableJudgement.cs

```diff
@@ -107,5 +111,18 @@ private Color4 judgementColour(HitResult judgement)
 
             return Color4.White;
         }
+
+        private readonly Vector4 baseFastColor = new Vector4(0.5f, 0.0f, 1.0f, 1);
+        private readonly Vector4 maxFastColor = new Vector4(0.5f, 1.0f, 0.5f, 1);
+        private readonly Vector4 baseSlowColor = new Vector4(0.0f, 0.0f, 1.0f, 1);
+        private readonly Vector4 maxSlowColor = new Vector4(0.0f, 1.0f, 0.5f, 1);
+        protected Color4 JudgementColour(Judgement judgement)
```

Comment:
`else if` and `return Color4.White` are redundant.

`protected Color4 JudgementColour(Judgement judgement) => judgement.TimeOffset >= 0 ? Color4.FromHcy(baseSlowColor + (maxSlowColor - baseSlowColor) * (float)(judgement.TimeOffset / JudgedObject.HitObject.HitWindows.HalfWindowFor(HitResult.Meh))) : Color4.FromHcy(baseFastColor + (maxFastColor - baseFastColor) * (float)(-judgement.TimeOffset / JudgedObject.HitObject.HitWindows.HalfWindowFor(HitResult.Meh)));`

Reply:
also, I would place a blank line between readonly fields and this method